opsimulator/deposits.go

```diff
@@ -36,7 +36,7 @@ type LogSubscriber interface {
 }
 
 // transforms Deposit event logs into DepositTx
-func SubscribeDepositTx(ctx context.Context, logSub LogSubscriber, depositContractAddr common.Address, ch chan<- *types.DepositTx) (ethereum.Subscription, error) {
+func SubscribeDepositTx(ctx context.Context, logSub LogSubscriber, depositContractAddr common.Address, ch chan<- *types.DepositTx, chLog chan<- *types.Log) (ethereum.Subscription, error) {
```

Comment:
high level it would be cleaner to just pass a single channel here. 

if you need other info from the depositTx like some of the log data, you can create a wrapper that includes the deposit tx along with the log

Reply:
update: sticking to the single channel